doc/build-unix.md

```diff
@@ -124,6 +124,24 @@ libqrencode (optional) can be installed with:
 Once these are installed, they will be found by configure and a bitcoin-qt executable will be
 built by default.
 
+Dependency Build Instructions: Fedora
+-------------------------------------
+Build requirements:
+
+    sudo dnf install libtool make autoconf automake openssl-devel libevent-devel boost-devel libdb4-devel libdb4-cxx-devel
```

Comment:
You're technically correct, just it's never listed in any build requirements because it's assumed to be in the minimum build requirements of anything in Fedora.